src/shared/services/transform/index.js

```diff
@@ -0,0 +1,15 @@
+export const formatWeekOrWorkdays = (days, isCalendarDays) => {
```

Comment:
`transform` is a very generic name.
Why not move it to `src/shared/services/date-utils/index.js`?

Reply:
Did not know we had that one, done ;)